manifests/server/contrib.pp

```diff
@@ -8,19 +8,17 @@
   String $package_name      = $postgresql::params::contrib_package_name,
```

Comment:
This should be `Optional[String[1]]` so `undef` is accepted.
```suggestion
  Optional[String[1]] $package_name = $postgresql::params::contrib_package_name,
```

Reply:
Fixed f57f823